### ISSUE TITLE
Use of uninitialized value in subroutine entry warnings

### ISSUE THREAD

**Issue by zpl** [tokuhirom/HTML-TreeBuilder-LibXML#10]
Hi.

i got 
Use of uninitialized value in subroutine entry at /Users/dan/perl5/perlbrew/perls/perl-5.20.0/lib/site_perl/5.20.0/HTML/TreeBuilder/LibXML/Node.pm line 160.

warning.

there is code to reproduce.

```perl
use strict;
use warnings;

use HTML::TreeBuilder::LibXML;

my $doc = <<'DOC';
<html>
    <body>
        <div>
            <div>a</div>
            <div>b</div>
        </div>
    </body>
</html>
DOC

my $tree = HTML::TreeBuilder::LibXML->new;
$tree->parse($doc);
$tree->eof;

my @nodes = $tree->findnodes("//body/div");

my $new_node = $nodes[0]->clone;
```

thanks.

**Comment by vividsnow** [comment]
please, release it on CPAN

**Comment by tokuhirom** [comment]
@vividsnow sorry! I forgot to release the module!
I shipped 0.25 on CPAN!

**Comment by vividsnow** [comment]
thanks!
